frame/system/src/lib.rs

```diff
@@ -1375,3 +1377,14 @@ impl<T: Trait> Lookup for ChainContext<T> {
 		<T::Lookup as StaticLookup>::lookup(s)
 	}
 }
+
+/// An implementation of `sp_runtime::traits::Block` to be used in test.
+pub type MockBlock<T> = generic::Block<
```

Comment:
This does not seem used to me, else why not feature gate it to test or at least std?

Reply:
in this following PR https://github.com/paritytech/substrate/compare/gui-automatic-construct-runtime...gui-automatic-construct-runtime-usage?expand=1
I used it a lot for pallet mocks.

I guess we could feature get but I don't see strong reason not to have in no_std as well. some people could make use of it for no_std mock.